### PR TITLE
Add option to remove clearly-defined lookup

### PR DESCRIPTION
**Pull request by JeroenKnoops**
Add option to disable ClearlyDefinedHarvester

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Some useful command line parameters are:
 - `--server.port=9090` changes the http port (from default 8080) to 9090.
 - `--bom-base.scan-licenses=false` disables the source code license scanner,
   reducing the machine load during development and testing.
+- `--bom-base.use-clearly-defined=false` disables the clearly-defined lookup,
```
Comment (timovandeput) [inline README.md:71]: Wondering if `clearly-defined=false` would be more in line with the `scan-licenses=false` option.

(My suggestion is to drop the "use-" prefix.)

File: README.md [hunk 2]
```diff
@@ -68,6 +68,8 @@ Some useful command line parameters are:
...
+  for debug purposes when developing harvesters.
```
Comment (timovandeput) [inline README.md:72]: I think it is just a switch to disable ClearlyDefined for whatever reason, and not just for debugging. (e.g. if ClearlyDefined is discontinued by Microsoft.)

(My suggestion is to drop this explicit reason.)

File: src/main/java/com/philips/research/bombase/ConfigProperties.java [hunk 3]
```diff
@@ -10,13 +10,23 @@
 @ConfigurationProperties(prefix = "bom-base")
 public class ConfigProperties {
     private boolean scanLicenses = true;
+    private boolean useClearlyDefined = true;
 
     public boolean isScanLicenses() {
         return scanLicenses;
     }
+    
+    public boolean isUseClearlyDefined() {
```
Comment (timovandeput) [inline src/main/java/com/philips/research/bombase/ConfigProperties.java:19]: Is this method about "is" or "use" ? 

(My suggestion is to make the name better express the intention.)

**Review by timovandeput — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JeroenKnoops) [reply README.md]: use `harvest-clearly-defined` now.

File: src/main/java/com/philips/research/bombase/ConfigProperties.java (hunk 3 above)
Comment (JeroenKnoops) [reply src/main/java/com/philips/research/bombase/ConfigProperties.java]: Changed it into `harvestClearlyDefined()`

File: README.md (hunk 2 above)
Comment (JeroenKnoops) [reply README.md]: Removed reason

**Review by timovandeput — APPROVED** [review]

**Pull request merged**